docs/opcodes/FD.mdx

```diff
@@ -5,18 +5,30 @@ group: System operations
 
 ## Notes
 
-Equivalent to [CREATE](/#F0), except the salt allows to create at a deterministic address.
+Abort execution, undo all state changes, return a value and refund the remaining gas to the caller.
+The error message returned is available to the caller in the returndata buffer.
```

Comment:
Not "error message", but rather "The [return data](/about) of the calling context is set as the given chunk of memory of this context."

Reply:
Change it